### ISSUE TITLE
Schema field default resolver crashes on some hash_key values

### ISSUE THREAD

**Issue by dpkp** [rmosolgo/graphql-ruby#1978]
ruby-2.5.1 / graphql-1.8.9

The schema default_resolver crashes if a field uses a non-standard hash_key, like so:

```
field :foo, String, hash_key: :"foo/bar"
field :bar, String, hash_key: "2nd test case"
```

```
SyntaxError (/usr/local/rvm/gems/ruby-2.5.1/gems/graphql-1.8.9/lib/graphql/schema/member/has_fields.rb:121: syntax error, unexpected '/', expecting ';' or '\n'
            def foo/bar(**args)
                   ^
/usr/local/rvm/gems/ruby-2.5.1/gems/graphql-1.8.9/lib/graphql/schema/member/has_fields.rb:124: syntax error, unexpected keyword_end, expecting end-of-input
            end
            ^~~):
```

**Comment by swalkinshaw** [comment]
This could potentially be fixed by https://github.com/rmosolgo/graphql-ruby/pull/1961. Never thought about this use case, but it's an interesting one.

**Comment by dpkp** [comment]
I put up a small fix for this specific issue here: #1979 . Feel free to reject that in favor of a larger change in #1961 for example. If so, perhaps add some test coverage for this use case?

**Comment by rmosolgo** [comment]
Fixed by #1961 :tada:
